File: WorkoutTracker/Models/SetResult.cs
using System;
namespace WorkoutTracker.Models
{
    public class SetResult
    {
        public int Id { get; set; }
        public int Order { get; set; }
        public string Notes { get; set; }
        public int Weight { get; set; }
        public int RepsActual { get; set; }
        public int RepsTargetLow { get; set; }
        public int RepsTargetHigh { get; set; }
        public bool ToFailure { get; set; }
        public int RestTime { get; set; }

        public int WorkoutSessionId { get; set; }
        public WorkoutSession WorkoutSession { get; set; }

        public int ExerciseId { get; set; }
        public Exercise Exercise { get; set; }
    }
}

File: WorkoutTracker/ClientApp/src/app/models/set-result.ts
export class SetResult {

  id: number;
  notes: string;
  order: number;
  weight: number;
  repsActual: number;
  repsTargetLow: number;
  repsTargetHigh: number;
  toFailure: boolean;
  restTime: number;
  workoutSessionId: number;
  exerciseId: number;

  constructor(order: number, weight: number, repsActual: number,
    repsTargetLow: number, repsTargetHigh: number, toFailure: boolean,
    restTime: number, workoutSessionId: number, exerciseId: number) {
    this.order = order;
    this.weight = weight;
    this.repsActual = repsActual;
    this.repsTargetLow = repsTargetLow;
    this.repsTargetHigh = repsTargetHigh;
    this.toFailure = toFailure;
    this.restTime = restTime;
    this.workoutSessionId = workoutSessionId;
    this.exerciseId = exerciseId;
  }

}